docs/src/main/sphinx/release/release-408.md

```diff
@@ -0,0 +1,71 @@
+# Release 408 (22 Feb 2023)
+
+## General
+
+* Fix query failure when performing a remote partitioning exchange on a
+  [structural type](structural-data-types) that contains `NULL` elements. ({issue}`16140`)
+
+## Security
+
+* Deprecate the `http-server.authentication.oauth2.groups-field` configuration 
```

Comment:
What's the functionality that's being deprecated? The entry should talk about that, instead.